<?xml version="1.0" encoding="UTF-8"?>
<!-- Path: gui\informacao_produto\deletion_confirmation_dialog.ui -->
<ui version="4.0">
 <class>DeletionConfirmationDialog</class>
 <widget class="QDialog" name="DeletionConfirmationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Confirmar Exclusão</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="warningIconLabel">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>icons/warning.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="warningLabel">
     <property name="styleSheet">
      <string notr="true">color: #CF222E; font-weight: bold;</string>
     </property>
     <property name="text">
      <string>Você está prestes a excluir este item. Esta ação não pode ser desfeita.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="confirmLabel">
     <property name="text">
      <string>Digite o nome do item para confirmar:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="placeholderText">
      <string>Digite o nome para confirmar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="motivoLabel">
     <property name="text">
      <string>Informe o motivo da exclusão (obrigatório):</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="motivoTextEdit">
     <property name="placeholderText">
      <string>Informe o motivo detalhado para a exclusão deste item...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeletionConfirmationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeletionConfirmationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>